### PR TITLE
Proofread and add signing-in conditions to all Maia features

### PR DESCRIPTION
**Pull request by Yiyun333**
To address:
https://github.com/mendix/docs/issues/8817
https://github.com/mendix/docs/issues/8818

### CODE REVIEW

File: content/en/docs/refguide/modeling/mendix-ai-assistance/logic-recommender.md
Comment (arjan-h) [inline content/en/docs/refguide/modeling/mendix-ai-assistance/logic-recommender.md:1]: I would rephrase this 'to get better AI recommendations,' into 
'to get better AI-based recommendations,'

Comment (Yiyun333) [reply content/en/docs/refguide/modeling/mendix-ai-assistance/logic-recommender.md:1]: Sounds good! I've made the change here: https://github.com/mendix/docs/pull/8825/commits/e3ce7b9a08b51605beb2f9931c4de92ecf20b479.

**Comment by Yiyun333** [comment]
Agree to merge via Slack.
